CMakeLists.txt

```diff
@@ -584,6 +584,7 @@ add_subdirectory(src/pal/prebuilt/inc)
 
 add_subdirectory(src/debug/debug-pal)
 
+add_definitions(-DFEATURE_COLLECTIBLE_ALC)
```

Comment:
I am not sure whether we need this define. It made sense when this started and the same code was still building for desktop. It is not the case anymore.